spec/dummy/app/views/home/headers.rb

```diff
@@ -138,13 +135,28 @@ def main
 
     hr
 
-    docs 'Basic Page Header', %{
+    h3 'Basic Page Header'
+
+    docs 'Multi-Step Flows', %{
+      div(class: 'page_header') {
+        div(class: 'page_header_context'){
+          span '2015 Civic Innovation Conference', href: '#'
```

Comment:
We should use a `beforeunload` warning for that. If we follow this logic, wouldn't we remove all links from the page?

(Also, the `href` attribute on a `span` tag is invalid...)